dev/reference/transactions/transaction-isolation.md

```diff
@@ -57,18 +57,13 @@ The MySQL Repeatable Read isolation level is not the snapshot isolation level. T
 
 ## Transaction retry
 
-Transactions that fail may automatically be retried by TiDB, which may lead to lost updates. This feature can be disabled by setting `tidb_retry_limit = 0`.
+Failed transactions might be automatically retried by TiDB, which can lead to update loss. You can enable **this feature** by setting `tidb_disable_txn_auto_retry = 0`.
 
-```
-[performance]
-...
-# The maximum number of retries when commit a transaction.
-retry-limit = 10
-```
+The value of `tidb_retry_limit` cannot be 0. Otherwise, the automatic retry is also disabled.
 
 ## Transactional anomalies caused by automatic retries
 
-Because TiDB automatically retries transactions [by default](#transaction-retry), the final result might not be as expected if the transactions created by the explicit `BEGIN` statement automatically retry after meeting a conflict.
+TiDB does not automatically retry transactions [by default](#transaction-retry). If automatic transaction retry is enabled, the final result might not be as expected if the transaction created by the explicit `BEGIN` statement automatically retries after encountering a conflict.
```

Comment:
This is another case of BEGIN|START TRANSACTION. Even `set autocommit=0` will also explicitly create a transaction, so I suggest we remove it here. If it is important to the original author, we could also say "non-autocommit transactions", which would be more accurate.

```suggestion
TiDB does not automatically retry transactions [by default](#transaction-retry). If automatic transaction retry is enabled, the final result might not be as expected if the transaction automatically retries after encountering a conflict.
```